conjur-oss/README.md

```diff
@@ -104,6 +104,39 @@ $  helm install \
    ./conjur-oss
 ```
 
+### Configuring Conjur Accounts
+
+By setting `account.create` to `true`, you can direct your Conjur
+container to create an admin account during startup. To retrieve the credentials
```

Comment:
I think `create an admin account` should be `create an account`, I'm not aware of different types of accounts.